src/items/class-features/a_few_more_steps!_(ex).json

```diff
@@ -37,7 +37,7 @@
       "gmnotes": "",
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p><sub><img src=\"systems/sfrpg/images/cup/gameplay/language.webp\" alt=\"\" width=\"25\" height=\"25\">&nbsp;</sub>Language-dependent <sub><img src=\"systems/sfrpg/images/cup/gameplay/mind.webp\" alt=\"\" width=\"25\" height=\"25\"></sub><span> </span>Mind-Affecting<sub><img src=\"systems/sfrpg/images/cup/gameplay/sense.webp\" alt=\"\" width=\"25\" height=\"25\"></sub><span> </span>Sense-Dependent</p>\n<p>As a reaction when an ally within 60 feet of you is reduced to 0 HP and would become unconscious, you can compel them to stay up and stagger out of the fray. The ally remains conscious, becomes @UUID[Compendium.sfrpg.conditions.LoRydw1D4bqlbkA2]{nauseated}, and gains temporary Hit Points equal to your envoy level + your Charisma modifier until the end of their next turn. Afterward, if the ally has 0 Hit Points, they become unconscious and dying as normal, including losing 1 Resolve Point at the end of their turn. Once a creature has been affected by this improvisation, they’re immune to it for 24 hours.</p>\n<p>At 6th level, the ally becomes @UUID[Compendium.sfrpg.conditions.aAQ6SW6iK0EVq8Ce]{staggered} and @UUID[Compendium.sfrpg.conditions.yUtkoIshf4dPruNd]{off-target} rather than nauseated. If the ally is stable and unconscious and spends 1 Resolve Point to regain 1 HP within 1 minute of falling unconscious, they instead regain HP equal to your envoy level + your Charisma modifier.</p>"
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p><sub>&nbsp;</sub>Language-dependent Mind-Affecting<span> </span>Sense-Dependent</p>\n<p>As a reaction when an ally within 60 feet of you is reduced to 0 HP and would become unconscious, you can compel them to stay up and stagger out of the fray. The ally remains conscious, becomes @UUID[Compendium.sfrpg.conditions.LoRydw1D4bqlbkA2]{nauseated}, and gains temporary Hit Points equal to [[@classes.envoy.levels+@abilities.cha.mod]]{your envoy level + your Charisma modifier} until the end of their next turn. Afterward, if the ally has 0 Hit Points, they become unconscious and dying as normal, including losing 1 Resolve Point at the end of their turn. Once a creature has been affected by this improvisation, they’re immune to it for 24 hours.</p>\n<p>At 6th level, the ally becomes @UUID[Compendium.sfrpg.conditions.aAQ6SW6iK0EVq8Ce]{staggered} and @UUID[Compendium.sfrpg.conditions.yUtkoIshf4dPruNd]{off-target} rather than nauseated. If the ally is stable and unconscious and spends 1 Resolve Point to regain 1 HP within 1 minute of falling unconscious, they instead regain HP equal to [[@classes.envoy.levels+@abilities.cha.mod]]{your envoy level + your Charisma modifier}.</p>"
```

Comment:
```suggestion
      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p>@Icon[type:language-dependent] @Icon[type:mind-affecting] @Icon[type:sense-dependent]</p>\n<p>As a reaction when an ally within 60 feet of you is reduced to 0 HP and would become unconscious, you can compel them to stay up and stagger out of the fray. The ally remains conscious, becomes @UUID[Compendium.sfrpg.conditions.LoRydw1D4bqlbkA2]{nauseated}, and gains temporary Hit Points equal to [[@classes.envoy.levels+@abilities.cha.mod]]{your envoy level + your Charisma modifier} until the end of their next turn. Afterward, if the ally has 0 Hit Points, they become unconscious and dying as normal, including losing 1 Resolve Point at the end of their turn. Once a creature has been affected by this improvisation, they’re immune to it for 24 hours.</p>\n<p>At 6th level, the ally becomes @UUID[Compendium.sfrpg.conditions.aAQ6SW6iK0EVq8Ce]{staggered} and @UUID[Compendium.sfrpg.conditions.yUtkoIshf4dPruNd]{off-target} rather than nauseated. If the ally is stable and unconscious and spends 1 Resolve Point to regain 1 HP within 1 minute of falling unconscious, they instead regain HP equal to [[@classes.envoy.levels+@abilities.cha.mod]]{your envoy level + your Charisma modifier}.</p>"
```